src/st/no3/3FF00.c

```diff
@@ -0,0 +1,1208 @@
+#include "no3.h"
+
+void EntityRoomTransition2(Entity* self) {
+    Entity* newEntity;
+    Primitive* prim;
+    // Used as both a loop variable and a temp
+    s32 localVar;
+
+    // These are weird fake vars needed for matching
+    Entity* gents = g_Entities;
```

Comment:
```suggestion
Entity* player = &PLAYER;

Reply:
This doesn't work though because on line 100 we have ` gents = &g_Entities[192];` - I definitely don't want to name the variable `player`, because then that line would have to say `player = &g_Entities[192]` which doesn't make much sense.